plugins/tiddlywiki/dynannotate/examples/searching-in-tiddlywiki.tid

```diff
@@ -0,0 +1,19 @@
+title: $:/plugins/tiddlywiki/dynannotate/searching-in-tiddlywiki
```

Comment:
I think this tiddler is just here as test data. It is a bit confusing that that cannot be determined by reading it. It would be clearer if the text of it were titled `$:/plugins/tiddlywiki/dynannotate/example-tiddler` tiddler and the content explicitly said "This tiddler is test data for use in the Dynannotate demos".

Reply:
I'm open to use something completely different here, some text unrelated to TW maybe. Like in the _simple_ examples. I did want to transclude a "real" tiddler, since it's a demonstration of the view template. Initially I _did_ just transclude "Searching in TiddlyWiki", but then I realized that this tiddler won't exist for most people who install the plugin in their wikis, so I made a copy of it...
I'm not really married to the idea, so if you have a better example (which preferrably uses a bunch of features of WikiText)...

Reply:
Maybe we can transclude the control panel basic tab?

Reply:
I used the tiddler fields tab from _Advanced_, because it's not interactive und thus not potentially confusing. Wouldn't someone to mess up their wiki because they think they're just playing with an example...